apps/api/src/app/testing/billing/webhook.e2e-ee.ts

```diff
@@ -135,14 +136,18 @@ describe('Stripe webhooks', () => {
         metered: { id: 'metered_subscription_id' },
       } as any);
       updateCustomerStub = sinon.stub(stripeStub.customers, 'update').resolves({});
+      // TODO: remove this after IS_IMPROVED_BILLING_ENABLED flag is removed
+      const IS_IMPROVED_BILLING_ENABLED = false;
+      getFeatureFlagStub = sinon.stub(GetFeatureFlag.prototype, 'execute').resolves(IS_IMPROVED_BILLING_ENABLED);
```

Comment:
Change under this feature flag will not affect this test so the value here doesn't matter